frontend/css/stats.less

```diff
@@ -112,3 +112,13 @@
   grid-template-columns: repeat(auto-fit, minmax(160px, max-content));
   place-content: space-evenly;
 }
+
+.custom-tooltip-genre-stats {
+  background: white;
+  padding: 12px;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.2);
+  text-align: center;
+  opacity: 0.9;
+}
```

Comment:
We just last week merge some changes to our CSS preprocessing, moving from Less to Sass.

Thankfully the solution is simple to resolve the merge conflict: move thes changes to the new [stats.scss file](https://github.com/metabrainz/listenbrainz-server/blob/master/frontend/css/sass/stats.scss) and remove the changes from this file (will be deleted after the merge)